modules/Utils/Strings.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Utils;
```

Comment:
Did you see the modules README?

```
This directory is meant to hold PHP / js files for inclusion in your
<gamename>.game.php / gamename.js if needed.

It is recommended to name your files starting with a trigram of your game name.
For example if you have a class to manage pathfinding in the game
Tobago, it will be called TBGPathFinding.php (and the class inside be
called TBGPathFinding) to avoid potential namespace collisions.
```

Let's chat about this.

Reply:
I've updated it to `Innovation` with namespaces, but we're pending the answer to https://discord.com/channels/753304735615811584/753750063725150289/973041581349285980 to see if we need to add some kind of improved syncing+renaming tooling here to handle this more fluidly for forks.